src/main/scala/org/allenai/pnp/semparse/SemanticParser.scala

```diff
@@ -368,7 +368,7 @@ class SemanticParser(val actionSpace: ActionSpace, val vocab: IndexedList[String
         currentScope.getVariableTemplates(curType) ++
         entityLinking.getEntitiesWithType(curType).map(_.template)
       val matches = templates.filter(_.matches(expIndex, exp, typeMap))
-      if (matches.size != 1) {
+      if (matches.size < 1) {
```

Comment:
This is wrong. matches.size must == 1. If it doesn't, there's something wrong with the logical forms or the action space.